ott/tools/plot.py

```diff
@@ -107,6 +126,41 @@ def barycenters(arg: Union[transport.Transport, jnp.ndarray],
 
   if isinstance(arg, transport.Transport):
     ot = arg
-    return _barycenters(ax, ot.geom.y, ot.a, ot.b, ot.matrix, **kwargs)
-
-  return _barycenters(ax, arg, a, b, matrix, **kwargs)
+    _barycenters(ax, ot.geom.y, ot.a, ot.b, ot.matrix, **kwargs)
+    return ax
+
+  _barycenters(ax, arg, a, b, matrix, **kwargs)
+  return ax
+
+
+def animate(image_fns: List[str],
+            gif_fn: str,
+            duration: int = 250,
+            loop: int = 0):
+  """Makes an animated GIF from list of images."""
+  if not PIL_is_importable:
+    warnings.warn(
+        'Pillow was not imported successfully. Doing nothing instead.',
+        RuntimeWarning,
+    )
+    return
+  imgs = [Image.open(_) for _ in image_fns]
+  imgs[0].save(gif_fn,
```

Comment:
i would rather stick to matplotlib.animation.FuncAnimation instead. This can be save on disk as gif (provided imagemagick is installed) and can be just used in a notebook without saving to disk. This is more natural in matplotlib and does not rely on writing to /reading from disk

Reply:
(see comments)